### ISSUE TITLE
Setup?

### ISSUE THREAD

**Issue by mooncreative** [derweili/fb-messenger-bot-for-woocommerce#7]
Hello, 

I am confident I have set up the API credentials correctly in the plugin and on fb developer, but I am not seeing my URL in the whitelisted area, and I'm also not seeing the checkboxes on the checkout page. 

I set this up about 30 minutes ago ... should I wait longer? I am logged in to Facebook, I'm the app developer, and the app is not in development mode. I have submitted the app for review, but haven't heard back yet. Should the plugin work before I submit it for review?

https://www.vapewellsupply.com/checkout/

**Comment by mooncreative** [comment]
Hello? Any help for whitelisting urls? Did I maybe do my app wrong in Developer?

**Comment by derweili** [comment]
Did you try to resave your setting? Every time you save the plugin settings, the whitelisting should be done.
Do you have a active ssl certificate?

**Comment by mooncreative** [comment]
Yes, I have an active SSL certificate. Should I do something different?

**Comment by lazmo88** [comment]
Hi derweili! Amazing plugin, just what I was looking for :) I am having the same issue as mooncreative, I checked browser console and received following error: 

`Refused to display 'https://www.facebook.com/plugins/messenger_checkbox.php?allow_login=true&ap…checked=true&sdk=joey&size=xlarge&user_ref=XXXXXX' in a frame because an ancestor violates the following Content Security Policy directive: "frame-ancestors https://www.facebook.com".`

Any idea what is wrong? I am using SSL via CloudFlare.

Thanks in advance!

**Comment by derweili** [comment]
Hi, there is a bug with the whitelisting of the url.
Normaly the "whitelisting task" should be executed every time you view the settings page. Unfortunately I don't have the time to find and to fix this bug yet.

Try to resave the plugin settings several times and wait several minutes. This should solve the problem.
